### PR TITLE
Structured Data 2021 SQL update - add page counts

### PR DESCRIPTION
**Pull request by tunetheweb**
Tweaks for #2174 

- [x] classic_microformats_types.sql
- [x] dublin_core_types.sql
- [x] facebook_types.sql
- [x] jsonld_contexts.sql
- [x] jsonld_entities_relationships.sql
- [x] jsonld_max_depth.sql - changed to percentiles and added sample urls
- [x] jsonld_relationship_values.sql
- [x] jsonld_relationships.sql
- [x] jsonld_same_ases.sql
- [x] jsonld_types.sql
- [x] microdata_item_types.sql
- [x] microformats2_types.sql
- [x] open_graph_types.sql
- [x] present_types.sql
- [x] rdfa_prefixes.sql
- [x] rdfa_type_ofs.sql
- [x] rdfa_vocabs.sql
- [x] summary_count.sql
- [x] twitter_types.sql

### CODE REVIEW

**Comment by tunetheweb** [comment]
FYI @jono-alderson @GregBrimble

**Comment by tunetheweb** [comment]
@jono-alderson let me know when you've reviewed the results in the Sheet and will merge this.

Also please delete the "(old)" tabs in Sheets once you're happy with the new numbers to avoid confusion.

File: sql/2021/structured-data/rdfa_prefixes.sql [hunk 1]
```diff
@@ -5,7 +5,7 @@ RETURNS ARRAY<STRING>
 LANGUAGE js AS """
```
Comment (rviscomi) [inline sql/2021/structured-data/rdfa_prefixes.sql:5]:
```suggestion
LANGUAGE js AS r"""
```

File: sql/2021/structured-data/rdfa_prefixes.sql [hunk 2]
```diff
@@ -5,7 +5,7 @@ RETURNS ARRAY<STRING>
...
   try {
     rendered = JSON.parse(rendered);
-    const prefixRegExp = new RegExp(/(?<ncname>[^:]*):\s+(?<uri>[^\s]*)\s*/gm)
+    const prefixRegExp = new RegExp(/(?<ncname>[^:]*):\\s+(?<uri>[^\\s]*)\\s*/gm)
```
Comment (rviscomi) [inline sql/2021/structured-data/rdfa_prefixes.sql:8]:
```suggestion
    const prefixRegExp = new RegExp(/(?<ncname>[^:]*):\s+(?<uri>[^\s]*)\s*/gm)
```

File: sql/2021/structured-data/present_types.sql [hunk 3]
```diff
@@ -1,73 +1,35 @@
...
-  ) AS opengraph,
-  SUM(CASE
-      WHEN JSON_EXTRACT(structured_data, '$.structured_data') IS NOT NULL AND JSON_EXTRACT(structured_data, '$.log') IS NULL THEN 1
-      ELSE
-        0
-    END
-  ) AS total,
-  client
+  client,
+  COUNTIF(CAST(JSON_EXTRACT(structured_data, '$.structured_data.rendered.present.rdfa') AS BOOL) = TRUE) AS rdfa,
```
Comment (rviscomi) [inline sql/2021/structured-data/present_types.sql:5]: Nit
```suggestion
  COUNTIF(CAST(JSON_EXTRACT(structured_data, '$.structured_data.rendered.present.rdfa') AS BOOL)) AS rdfa,
```

**Review by rviscomi — APPROVED** [review]

File: sql/2021/structured-data/rdfa_prefixes.sql (hunk 2 above)
Comment (tunetheweb) [reply sql/2021/structured-data/rdfa_prefixes.sql:8]: I was confused about this too. Weirdly it wouldn’t rub for me on BigQuery without this. Complained about an illegal escape, which I’d not seen before and made me wonder why it worked for original analyst.

Maybe I had a weird setting enabled. Or was due to lack of `r` in `r”””`. Will give it another look.

Comment (tunetheweb) [reply sql/2021/structured-data/rdfa_prefixes.sql:8]: Yeah it was the lack of `r"""` - TIL!

**Review by rviscomi — APPROVED** [review]

**Pull request merged**